pkg/serverless/trace/inferredspan/span_enrichment.go

```diff
@@ -217,6 +219,105 @@ func (inferredSpan *InferredSpan) EnrichInferredSpanWithS3Event(eventPayload eve
 	}
 }
 
+type rawTraceContext struct {
+	TraceID  string `json:"x-datadog-trace-id"`
+	ParentID string `json:"x-datadog-parent-id"`
+}
+
+type convertedTraceContext struct {
+	TraceID  *uint64 `json:"x-datadog-trace-id"`
```

Comment:
We don't need json serialization declarations if we're not serializing this to JSON, so unless we're reading _from_ json or serializing _to_ json, we can remove the `json:"...` bits